runtime/doc/starting.txt

```diff
@@ -1392,6 +1392,8 @@ Neovim will look for (and auto-create) in the various XDG parent directories.
 For example, setting `$NVIM_APPNAME` to "neovim" before running Neovim will
 result in Neovim looking for configuration files in `$XDG_CONFIG_HOME/neovim`
 instead of `$XDG_CONFIG_HOME/nvim`.
+`$NVIM_APPNAME` should not contain any path separators or characters that are
```

Comment:
```suggestion
`$NVIM_APPNAME` must not contain any path separators or characters that are
```

Alternatively I'd even just leave this out of the docs. It's guarded by an explicit check and it's obvious.

Reply:
Very good point! Done.